sw/SoC/examples/blinky/Makefile

```diff
@@ -28,14 +30,16 @@ STARTUP = 	      startup
 # OBJDUMP = $(RV_PREFIX)objdump $(32BITODFLAGS)
 #
 
-RV_PREFIX = 	riscv32-unknown-elf-
+RV_PREFIX = 	$(RV_TOOLCHAIN_PATH)
 CC = 			$(RV_PREFIX)gcc
 LD = 			$(RV_PREFIX)ld
 OBJDUMP = 		$(RV_PREFIX)objdump
 OBJCOPY = 		$(RV_PREFIX)objcopy
 
-CFLAGS = 		-march=rv32g_zicsr_zifencei -O0 -c
-LDFLAGS = 		-nostdlib -T$(LD_SCRIPT)
+CFLAGS = 		-march=rv32ima_zicsr_zifencei -O0 -c 
```

Comment:
This could not work in every cross-compiler installation, for example I got an error if the D is not present